### ISSUE TITLE
Since M$ April update ver 1803 the Monitor not working

### ISSUE THREAD

**Issue by germanmember** [IAmPicard/STTVoyage#4]
It crashed if i logged in.
Without the M$ Update it works (tested on a notebook).

**Comment by IAmPicard** [comment]
Yes, Creators Update broke notifications for non-store apps, which effectively renders the tool useless. Sorry, not sure what to do here, I'm probably going to deprecate the tool and add some / most of its features into the [desktop tool](https://github.com/IAmPicard/StarTrekTimelinesSpreadsheet).
